src/ucp/tag/rndv.c

```diff
@@ -438,8 +438,8 @@ static ucs_status_t ucp_rndv_progress_send(uct_pending_req_t *self)
                                         ucp_rndv_pack_single_data);
     } else {
         /* send multiple bcopy messages (fragments of the original send message) */
-        ucs_trace_data("send on sreq %p, am lane: %d. multi message (bcopy)",
-                       sreq, sreq->send.lane);
+        ucs_trace_data("send on sreq %p, am lane: %d, offset: %zu. multi message (bcopy)",
+                       sreq, sreq->send.lane, sreq->send.state.offset);
```

Comment:
sure, i'll add both. 
(only contig datatype is currently supported for rndv on the sender's side)